python/tests/wrapper/test_05_multimodal.py

```diff
@@ -0,0 +1,215 @@
+    rus.append(u'Тематическая модель статистический анализ текст коллекция')
+
+    ens.append(u'LDA statistical topic model text collection')
+    rus.append(u'LDA статистический тематическая модель текст документ коллекция')
+
+    ens.append(u'PLSA statistical analysis text model')
+    rus.append(u'PLSA статистический анализ документ текст модель')
+
+    ens.append(u'ARTM analysis topic model')
+    rus.append(u'ARTM анализ документ topic model')
+
+    ens.append(u'Pinnipeds seal marine mammal order')
+    rus.append(u'Тюлень семейство млекопитающие моржовых отряд ласто ногие')
+
+    ens.append(u'Eared seal marine predatory mammal')
+    rus.append(u'Ушастый тюлень семейство млекопитающие отряд хищный семейство моржовых ласто ногие')
+
+    ens.append(u'Eared Crocodilia predatory reptile not mammal')
+    rus.append(u'Ушастый крокодил гена отряд хищный не млекопитающие коротко ногие')
+
+    ru_dic = {}  # mapping from russian token to its index in batch.token list
+    en_dic = {}  # mapping from english token to its index in batch.token list
+    batch = messages.Batch()  # batch representing the entire collection
+    batch.id = str(uuid.uuid1())
+    dict_config = messages.DictionaryConfig()  # BigARTM dictionary to initialize model
+
+    def append(tokens, dic, item, class_id):
+        for token in tokens:
+            if not dic.has_key(token):              # New token discovered:
+                dic[token] = len(batch.token)       # 1. update ru_dic or en_dic
+                batch.token.append(token)           # 2. update batch.token and batch.class_id
+                batch.class_id.append(class_id)
+                entry = dict_config.entry.add()   # 3. update dict_config
+                entry.key_token = token
+                entry.class_id = class_id
+
+            # Add token to the item.
+            item.field[0].token_id.append(dic[token])
+            # replace '1' with the actual number of token occupancies in the item
+            item.field[0].token_count.append(1)
+
+    # Iterate through all items and populate the batch
+    for (en, ru) in zip(ens, rus):
+        next_item = batch.item.add()
+        next_item.id = len(batch.item) - 1
+        next_item.field.add()
+        append(string.split(ru.lower()), ru_dic, next_item, russian_class)
+        append(string.split(en.lower()), en_dic, next_item, english_class)
+
+    batches_folder = tempfile.mkdtemp()
+    try:
+        # Create the instance of low-level API and helper object
+        lib = artm.wrapper.LibArtm()
+        helper = helpers.TestHelper(lib)
+
+        # Save batch and dictionary on the disk
+        lib.ArtmSaveBatch(batches_folder, batch)
+
+        dict_config.name = dictionary_name
+        with open(os.path.join(batches_folder, dictionary_name), 'wb') as file:
+            file.write(dict_config.SerializeToString())
+
+        # Create master component and scores
+        sp_phi_score_rus = messages.SparsityPhiScoreConfig()
```

Comment:
Is there a pythonic way to do this in one line?
```
        sp_phi_score_rus = messages.SparsityPhiScoreConfig()
        sp_phi_score_rus.class_id = russian_class
```
In C# this can be done via initializers: `new SparsityPhiScoreConfig {class_id = russian_class}`. Ain't there something similar in python?